freshmaker/errata.py

```diff
@@ -318,7 +318,7 @@ def _rhel_release_from_product_version(self, errata_id, product_version):
 
         return data["rhel_release"]["name"]
 
-    def get_builds(self, errata_id, rhel_release_prefix=None):
+    def get_builds(self, errata_id, rhel_release_prefix=None, return_srpm=True):
         """
```

Comment:
Can we have a separate api to get binary rpms to avoid confusion?

Reply:
Well actually the whole functionality to filter 'srpms' or 'binary rpms' is located inside small function inside that **get_builds** method. So If we want to have separate method, it will have like 75% of code exactly the same as that method, so it will be a lot of duplication.